src/relayserver/penalizer/TransactionDataCache.ts

```diff
@@ -0,0 +1,70 @@
+import AsyncNedb from 'nedb-async'
```

Comment:
do we really need to implement a persistent cache ?
I thought of having an in-memory dictionary, and maybe with lru cleanup 

e.g. https://www.npmjs.com/package/lru-cache

Reply:
I think it is easy to do and will not hurt. Not undoing it right now.